index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <style>
+    #nav-bar {
+      display: flex;
+      flex-wrap: nowrap;
+      justify-content: space-between;
+      width: 80vw;
+      position: fixed;
+      top: 0;
+    }
+
+    @media (max-width: 600px) {
+      #header-img {
+        width: 150px;
+      }
+    }
+  </style>
```

Comment:
- [ ] I think moving the styling to a dedicated file, will make the code cleaner.

Reply:
- [ ]  Nicely done 👍